migration-test-empty.sh

```diff
@@ -7,7 +7,7 @@ curl -sL https://run.spawn.cc/install | sh > /dev/null 2>&1
 export PATH=$HOME/.spawnctl/bin:$PATH
 echo "spawnctl successfully installed"
 
-export SPAWN_EMPTY_IMAGE=postgres:empty
+export SPAWN_EMPTY_IMAGE=postgres-empty
```

Comment:
while we're here we could/should start going for deterministic behaviour so something like `postgres-empty:v13`. otherwise only lerd knows what version we'll get (hopefully latest tag is the newest but 🤷)